x-pack/plugins/cloud_security_posture/public/pages/rules/rules_counters.tsx

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+} from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { useParams } from 'react-router-dom';
+import { Chart, Partition, PartitionLayout, Settings } from '@elastic/charts';
+import { FormattedMessage } from '@kbn/i18n-react';
+import noDataIllustration from '../../assets/illustrations/no_data_illustration.svg';
+import { BenchmarksCisId } from '../../../common/types/benchmarks/v2';
+import { useCspIntegrationLink } from '../../common/navigation/use_csp_integration_link';
+import { useNavigateFindings } from '../../common/hooks/use_navigate_findings';
+import { cloudPosturePages } from '../../common/navigation/constants';
+import {
+  CSPM_POLICY_TEMPLATE,
+  KSPM_POLICY_TEMPLATE,
+  RULE_FAILED,
+  RULE_PASSED,
+} from '../../../common/constants';
+import { statusColors } from '../../common/constants';
+import { useCspBenchmarkIntegrationsV2 } from '../benchmarks/use_csp_benchmark_integrations';
+import { DASHBOARD_COUNTER_CARDS } from '../compliance_dashboard/test_subjects';
+import { CspCounterCard } from '../../components/csp_counter_card';
+import { useKibana } from '../../common/hooks/use_kibana';
+
+const EvaluationPieChart = ({ failed, passed }: { failed: number; passed: number }) => {
+  const {
+    services: { charts },
+  } = useKibana();
+
+  return (
+    <Chart size={{ height: 30, width: 30 }}>
+      <Settings
+        theme={[
+          {
+            partition: {
+              linkLabel: { maximumSection: Infinity, maxCount: 0 },
+              outerSizeRatio: 0.75,
+              emptySizeRatio: 0.7,
+            },
+          },
+        ]}
```

Comment:
This configuration seems specific and complex, I think It would be helpful to add comments explaining the reasoning behind these settings.

Reply:
i removed the linkLabel, i wanted to add a comment for the other props but found myself repeating their names, i think they are self explanatory